unbound variable EPOCHREALTIME

#### Describe the bug

hi there! great lib, I wanted to use it for monitoring some bash scripts behavior. however, running [the `README.md` traces sample](https://github.com/krzko/opentelemetry-shell#trace-example) ends up in an error in plain bash Docker runtime:

```shell
./spam.sh

 _____ _____ _____ _____ _____ _____ __    _____ _____ _____ _____ _____ __ __
|     |  _  |   __|   | |_   _|   __|  |  |   __|     |   __|_   _| __  |  |  |
|  |  |   __|   __| | | | | | |   __|  |__|   __| | | |   __| | | |    -|_   _|
|_____|__|  |_____|_|___| |_| |_____|_____|_____|_|_|_|_____| |_| |__|__| |_|
 _____ _____ _____ __    __
|   __|  |  |   __|  |  |  |
|__   |     |   __|  |__|  |__
|_____|__|__|_____|_____|_____|

2022-10-04 15:41:58 [INFO] [spam.sh#L43] [source()] Initialising OpenTelemetry Shell v0.0.8
/appl/otello/opentelemetry-shell/library/time.sh: line 34: EPOCHREALTIME: unbound variable
2022-10-04 15:41:58 [INFO] [spam.sh#L10] [sleep_for()] Sleeping for 1 sec...
/appl/otello/opentelemetry-shell/library/time.sh: line 34: EPOCHREALTIME: unbound variable
2022-10-04 15:41:59 [ERROR] [spam.sh#L41] [net_client_post()] The requested URL returned an error: https://[OMITTED]/v1/traces
```

it looks like `$EPOCHREALTIME` was [not expanded properly](https://github.com/krzko/opentelemetry-shell/blob/main/library/time.sh#L34).

#### To Reproduce

Steps to reproduce the behavior:

1. get some linux Docker image
2. clone the repository on `main` (btw you didn't tag `v0.0.8` yet)
3. copy paste the traces sample
4. run it

#### Expected behavior

since I do not have the `$EPOCHREALTIME` defined, it should use date instead.

#### Screenshots

see bash output above.

#### Environment

- a plain linux in Docker, having installed `curl`, `git`, `hostname` and `jq`

#### Additional context

-

--- a/library/time.sh
+++ b/library/time.sh
@@ -31,7 +31,7 @@ get_epoch_now() {
       log_debug "Using gdate..."
     fi
     epoch="$(gdate +%s.%N)"
-  elif [ "${EPOCHREALTIME}" ]; then
+  elif [ -n "${EPOCHREALTIME-}" ]; then
     if [ -z "${OTEL_LOG_LEVEL-}" ]; then
       log_debug "Using ${EPOCHREALTIME}..."
     fi

